tests/plugins/easyimage/manual/selectionafterpaste.md

```diff
@@ -0,0 +1,14 @@
+@bender-tags: 4.9.0, bug, 932
```

Comment:
Actually this time we can start to reference it using dedicated GH issue number, so it's going to be `1529` in this case.